doc/api/http.markdown

```diff
@@ -776,6 +776,10 @@ be called before [response.end()][] is called.
 If you call [response.write()][] or [response.end()][] before calling this, the
 implicit/mutable headers will be calculated and call this function for you.
 
+When headers have been set with `response.setHeader`, they will be merged with
+any headers passed to `request.writeHead`, with the headers passed to
+`request.writeHead` given precedence.
+
```

Comment:
+1 .. quick thought tho: a simple example might be worthwhile including here also... e.g. 
```js
// returns content-type = text/plain
const server = http.createServer((req,res)=>{
  res.setHeader('Content-Type', 'text/html');
  res.setHeader('X-Foo', 'bar');
  res.writeHead(200, {'Content-Type': 'text/plain'});
  res.end('ok');
});
```